modules/deployment-scripts/create-kv-certificate/create-kv-cert.sh

```diff
@@ -12,9 +12,9 @@
 
     if [ -z "$issuerName" ] || [ -z "$issuerProvider" ]; then
         policy=$(az keyvault certificate get-default-policy \
-            | sed -e s/CN=CLIGetDefaultPolicy/CN=${certCommonName}/g )
+            | sed -e s/CN=CLIGetDefaultPolicy/CN="${certCommonName}"/g )
     else
-      if [ "$issuerProvider" == "DigiCert" ] || [ "$issuerProvider" == "GlobalCert"]; then
+      if [ "$issuerProvider" == "DigiCert" ] || [ "$issuerProvider" == "GlobalCert" ]; then
         az keyvault certificate issuer create \
           --vault-name $akvName \
           --issuer-name $issuerName \
```

Comment:
📝 **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)